src/components/container/TermInformation/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from "./TermInformation";
```

Comment:
Minor: You could probably export `TermInformationReference` from here as well, to shorten the its imports a little.